napari/utils/triangulation_backend.py

```diff
@@ -31,12 +31,20 @@ class TriangulationBackend(StrEnum):
 
     def __str__(self) -> str:
         """Return the string representation of the backend."""
-        return self.name.replace('_', ' ').title()
+        return str(self.value)
```

Comment:
This seems like a step backward. I think it should be reverted, and it suggests to me that the changes from `auto()` were not needed...

Reply:
These changes comes from this line in preference dialog code:

https://github.com/napari/napari/blob/7fa8c789dc109d0075d273dd2fba13aae3aca20f/napari/_qt/dialogs/preferences_dialog.py#L201

So instead of using `str(enum)` we use `enum.value` for preparing dialog. I do not want to touch this in this PR...

Reply:
oooh, I thought this was in StrEnum, not in TriangulationBackend! Ok this is fine. 🙏